### PR TITLE
Deepcode - Fix the bugs #521

### PR DESCRIPTION
**Pull request by marcnet80**
https://github.com/erxes/erxes-api/issues/521

### CODE REVIEW

**Comment by batamar** [comment]
Hi @marcnet80. Thanks for the pr. But there are some lint errors. Please run yarn tslint and fix the following errors

s1

File: src/index.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import * as fs from 'fs';
 import { createServer } from 'http';
 import * as path from 'path';
 import * as request from 'request';
+import * as xss from 'xss';
```
Comment (batamar) [inline src/index.ts]: use import { filterXSS } from 'xss'

Comment (marcnet80) [reply src/index.ts]: replaced, thank you.

**Comment by marcnet80** [comment]
> Hi @marcnet80. Thanks for the pr. But there are some lint errors. Please run yarn tslint and fix the following errors
> 
> s1
renamed to `Path`.

**Comment by batamar** [comment]
@marcnet80 merged. thank you

**Pull request merged**